demo/webpage/ts/types.ts

```diff
@@ -2,6 +2,7 @@
 type RecursivePartial<T> = {
     [P in keyof T]?: RecursivePartial<T[P]>;
 };
+interface IIndexable<T = any> { [key: string]: T }
```

Comment:
Not a big deal but Is this an extra I

Reply:
No. When you see an extra "I" in front of a type it often means it is an interface